tests/system/src/test/java/test/javafx/scene/Snapshot3Test.java

```diff
@@ -0,0 +1,65 @@
+package test.javafx.scene;
```

Comment:
Copyright header should be added to all new files. You can copy the header from any other file ([header sample](https://github.com/openjdk/jfx/blob/20325e1c3ec4c4e81af74d3d43bf3a803dbe1a51/tests/system/src/test/java/test/javafx/scene/Snapshot2Test.java#L1).)
Only change would be that, this file would contain only year `2020` in the copyright header.